src/components/tests/Steps.test.tsx

```diff
@@ -1,24 +1,75 @@
 import React from 'react';
 import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
 
-import Steps from '../Steps';
+import Steps, {
+  TutureMenu,
+  TutureMenuItem,
+  TutureSteps,
+} from '../Steps';
 import { catalogs, catalogsInfo } from './utils/data';
 
-test('Steps render successfully', () => {
+describe('Steps component/dom/events level behave normally', () => {
```

Comment:
I guess the description here (and below) should be `Steps behave normally on component/dom/events level`, which sounds more of a normal English sentence.